### DISCUSSION TITLE
作者能不能 在设置里面添加一键更新所有播放列表歌曲的标签加载呀

### DISCUSSION THREAD

**Discussion by freedomxzr** [zhongyang219/MusicPlayer2#618] [Q&A]
我播放的列表很多 ，是通过脚本一键生成的，因此刚开始使用的时候 都要手动 把每个列表点一次 才能把 标签属性加载出来，能否加一个一键更新标签的功能？ 谢谢作者大大大~~~

**Comment by lrisora** [comment]
如果是指的歌曲元数据"标题","艺术家","唱片集"这些的话可以将你的音频文件所在目录添加到“媒体库目录”，
应用设置后点“刷新媒体库”就会自动加载这些路径下的所有音频元数据到媒体库（可能很慢）
把所有你希望播放的音乐所在文件夹（会自动遍历子文件夹）全加入“媒体库目录”也是媒体库的标准用法

如果指的是"曲目数","总时长"的话还没有，要点一下才会从媒体库计算（去读取文件很花时间所以媒体库还没有的按0计算）
如果需要的话或许可以加一个更新所有播放列表的"曲目数","总时长"右键菜单项，
我觉得这样有点鸡肋因为仍然是从媒体库读，不会在这里去读文件更新媒体库的
